### ISSUE TITLE
called `Option::unwrap()` on a `None` value

### ISSUE THREAD

**Issue by dannyadkins** [prisma/prisma#17688]
Hi Prisma Team! My Prisma Client just crashed. This is the report:
## Versions

| Name            | Version            |
|-----------------|--------------------|
| Node            | v18.13.0           | 
| OS              | darwin-arm64       |
| Prisma Client   | 4.8.1              |
| Query Engine    | d6e67a83f971b175a593ccc12e15c4a757f93ffe|
| Database        | mongodb            |

## Logs
```
prisma:tryLoadEnv Environment variables not found at null
prisma:tryLoadEnv Environment variables loaded from /Users/danieladkins/dev/collective-monorepo/packages/database/.env
prisma:tryLoadEnv Environment variables not found at null
prisma:tryLoadEnv Environment variables loaded from /Users/danieladkins/dev/collective-monorepo/packages/database/.env
prisma:client dirname /Users/danieladkins/dev/collective-monorepo/node_modules/prisma-governance
prisma:client relativePath ../../packages/database/prisma
prisma:client cwd /Users/danieladkins/dev/collective-monorepo/packages/database/prisma
prisma:client clientVersion 4.8.1
prisma:client clientEngineType library
prisma:client:libraryEngine internalSetup
prisma:client:libraryEngine:loader Searching for Query Engine Library in /Users/danieladkins/dev/collective-monorepo/.prisma/client
prisma:client:libraryEngine:loader Searching for Query Engine Library in /Users/danieladkins/dev/collective-monorepo/node_modules/prisma-governance
prisma:client:libraryEngine:loader loadEngine using /Users/danieladkins/dev/collective-monorepo/node_modules/prisma-governance/libquery_engine-darwin-arm64.dylib.node
prisma:client:libraryEngine library starting
prisma:client:libraryEngine library started
prisma:client:libraryEngine sending request, this.libraryStarted: true
```

## Client Snippet
```ts
    const proposals = await governance.proposal.findMany({
      where: filters,
      orderBy,
      skip: (page - 1) * limit,
      take: limit,
    });
```

## Schema
```prisma
model Proposal {
  id                   String                       @id @default(auto()) @map("_id") @db.ObjectId
  v                    Int?                         @map("__v")
  auctionId            String?
  blockchain           String
  calldatas            String[]
  creation             ProposalsCreation
  customFields         ProposalsCustomFields?
  description          String
  entityId             String
  lastUpdated          ProposalsLastUpdated?
  linkedPostId         String
  linkedSubprojectSlug String?
  metadata             ProposalsMetadata
  network              String
  nounsDynamicQuorum   ProposalsNounsDynamicQuorum?
  options              Json
  payoutAmount         ProposalsPayoutAmount?
  proposalId           String
  proposer             String
  signatures           String[]
  status               String
  strategy             ProposalsStrategy?
  targets              String[]
  title                String?
  tldr                 String?
  tokenContract        String?
  totalUniqueVotes     Int
  totalVotes           Int
  type                 String
  uniqueId             String                       @unique(map: "uniqueId_1")
  updatedAt            DateTime?                    @db.Date
  values               String[]
  profile              Profile?                     @relation(fields: [proposer], references: [address])

  @@index([entityId, proposalId], map: "entityId_1_proposalId_1")
  @@index([entityId], map: "entityId_1")
  @@index([payoutAmount.eth(sort: Desc)], map: "payoutAmount.eth_-1")
  @@map("proposals")
}
```

## Prisma Engine Query
```
{
      where: filters,
      orderBy,
      skip: (page - 1) * limit,
      take: limit,
    }
```

**Comment by laplab** [comment]
Hey @dannyadkins, can you please confirm if this issue still reproduces on the latest Prisma version 5.11.0? If yes, can you please provide:
1. `npx prisma version`
2. MongoDB version you are using
3. Full client query (without any variables in it)
4. Minimal dataset the issue is reproducible on

**Comment by laplab** [comment]
Hey @dannyadkins, we are going to close this issue for now, as we do not have enough information to reproduce it. If you continue experiencing the same problem, please feel free to open a new issue with the details requested above.
